crates/ibc/src/core/ics03_connection/msgs.rs

```diff
@@ -24,11 +24,11 @@ pub mod conn_open_try;
 
 /// Enumeration of all possible messages that the ICS3 protocol processes.
 #[derive(Clone, Debug, PartialEq, Eq)]
-pub enum ConnectionMsg {
-    ConnectionOpenInit(MsgConnectionOpenInit),
-    ConnectionOpenTry(MsgConnectionOpenTry),
-    ConnectionOpenAck(MsgConnectionOpenAck),
-    ConnectionOpenConfirm(MsgConnectionOpenConfirm),
+pub(crate) enum ConnectionMsg {
+    Init(MsgConnectionOpenInit),
```

Comment:
Can we change all these to `OpenInit`, `OpenTry`, etc

Reply:
Using `Open*` rust compiler clipy still holds errors.